Update Composition docs for Kilted (#5487)
ros2/ros2_documentation: source/Tutorials/Intermediate/Composition.rst

>    $ ros2 component load /ComponentManager composition composition::Talker
>    $ ros2 component load /ComponentManager composition composition::Listener

Reviewer: ```suggestion
   $ ros2 component load /ComponentManager composition composition::Talker
   Loaded component 1 into '/ComponentManager' container node as '/talker'
   $ ros2 component load /ComponentManager composition composition::Listener
   Loaded component 2 into '/ComponentManager' container node as '/listener'
```

> You can get the unique ID of each component by just listing them:
> 
> .. code-block:: console
> 
>    $ ros2 component list
>    /ComponentManager
>      1  /talker
>      2  /listener
> 
> Use the unique ID to unload the component from the component container.

Reviewer: Using the `ros2 component list` command again is a good idea, but I would first rely on the `ros2 component load` output (see above) to get the ID.

```suggestion
The unique ID of a component is printed when it gets loaded.
You can also get the unique IDs of all components by just listing them now that they are loaded:

.. code-block:: console

   $ ros2 component list
   /ComponentManager
     1  /talker
     2  /listener

Use the unique ID to unload the component from the component container.
```

.. code-block:: console

   $ ros2 component load /ComponentManager image_tools image_tools::Cam2Image -p burger_mode:=true

>    $ ros2 run rqt_image_view rqt_image_view

Reviewer: Why is this being added? Even if it's using an image-related component, this example is just about passing parameter values, so I don't think this is needed.

Author: I just thought it would be nice to show it working / show that the parameter has passed properly.

Reviewer: It's a nice idea, but the thing is that you then have to explain what to look out for (i.e., what this parameter does), since this is not explained anywhere in this article. If you want to add this line, I'd add a simple comment with a very short description of what to look out for:

```suggestion
   $ ros2 run rqt_image_view rqt_image_view  # Shows that X is now Y
```

Author: Thanks!

Reviewer: Sorry, `X is now Y` was just a placeholder for an actual explanation :sweat_smile: could you figure out what setting `burger_mode:=true` does and describe it here instead of `X is now Y`?

Author: Oh! That's my bad, sorry! Fixed it in [7f546d5](https://github.com/ros2/ros2_documentation/pull/5487/commits/7f546d560928dc01c0c814ddff25c13c2d3dd0cd).

Reviewer: I was just not clear enough! Thanks for adding the comment!
